integration_tests/tests/forseti/controls/explain.rb

```diff
@@ -0,0 +1,375 @@
+/**
+* Copyright 2018 Google LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*      http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+require 'json'
+control 'explain' do
+
+    describe "Explain" do
+        before :context do
+            command("forseti inventory purge 0").result
+            command("forseti inventory create").result
+            inventory_id = JSON.parse(command("forseti inventory list").stdout).fetch("id")
+            command("forseti model create --inventory_index_id #{inventory_id} model_new").result
+            command("forseti model use model_new").result
+            command("sudo apt-get -y install mysql-client").result
+        end
+
+        describe "List members" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_members --prefix rdevani").stderr).to eq ""
+            end
+        end
+
+        describe "List IAM roles" do
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_roles --prefix roles/iam").stdout).to match /roles\/iam.organizationRoleAdmin/
+            end
+
+            it "should be visible from the command-line" do
+                expect(command("forseti explainer list_roles --prefix roles/iam").stdout).to match /roles\/iam.organizationRoleViewer/
```

Comment:
Can all the `list_role`  matching be consolidated?  Wouldn't the output be all the roles instead of just one?

Reply:
The only other way to consolidate the tests is to use a regular expression. The complex regex impacts the readability and is harder to maintain when new assertions needs to be added.